src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -164,8 +164,8 @@ public static void main(String... args) throws IOException {
       if (isEmptyCompilationUnit(target.getValue())) {
         boolean isASyntheticReturnType = addMissingClass.isASyntheticReturnType(target.getKey());
         boolean isASyntheticSuperClass =
-            !addMissingClass.getSuperClass().equals("")
-                && target.getKey().contains(addMissingClass.getSuperClass());
+            !addMissingClass.getSuperClass().isEmpty()
+                && addMissingClass.getSuperClass().contains(target.getKey());
```

Comment:
It's better style to avoid calling the same method twice in a conditional. In this case, you can abstract `addMissingClass.getSuperClass()` into a local variable on the line before.